arcgis-ios-sdk-samples/Augmented reality/Collect data in AR/README.md

```diff
@@ -18,24 +18,25 @@ When you tap, an orange diamond will appear at the tapped location. You can move
 
 ## How it works
 
-1. Create an `ArcGISARView`. Set the atmosphere effect to `none` and the space effect to `transparent`. Create and show a scene in the scene view.
+1. Create an `ArcGISARView`. Set the atmosphere effect to `none` and the space effect to `transparent`. Create and show a scene in the scene view. 
```

Comment:
The first bullet is not present in the code - the code does not change the atmosphere or space effect.

How should I handle the discrepancy between readme and implementation?

Reply:
If that is the case, the SDK-specific version of the code wins out, i.e. don't make any mention of changing the atmospheric effect here in the README.

Reply:
@philium I would propose that we do not include that line if the sample code does not show that being done.

Reply:
@mikewilburn I agree! I was just pointing out _why_ iOS doesn't include it when other platforms do.

Reply:
In the other example
https://github.com/Esri/arcgis-runtime-samples-ios/blob/6967286dbe41bd389f18b9de32b92c4374920a70/arcgis-ios-sdk-samples/Augmented%20reality/Explore%20scenes%20in%20flyover%20AR/ExploreScenesInFlyoverAR.swift#L80-L81
it sets those two properties explicitly, and the README includes that step. So for this one I think it is better to remove the line as they are not explicitly set.